crates/forge/src/main.rs

```diff
@@ -166,6 +167,17 @@ fn test_workspace(args: TestArgs) -> Result<bool> {
         bail!("Scarb build did not succeed")
     }
 
+    let test_build_output = Command::new("scarb")
```

Comment:
If you want to rely on scarb profile chose by user (line 149), you should also pass it to scarb on build cmd via `SCARB_PROFILE` env var. https://docs.swmansion.com/scarb/docs/writing-extensions/subcommands.html#environment-variables

Reply:
I guess every if every scarb process uses this env var it should be fine

Reply:
Yes, we do